src/python/grpcio_tests/tests_aio/unit/metadata_test.py

```diff
@@ -56,20 +59,12 @@
     ),
     (
         TypeError,
-        (({}, {}),),
+        ((None, {}),),
```

Comment:
The invalid metadata test is testing if our library is resistant against badly formed metadata. Maybe we should test both cases?

Reply:
Please take a look at the corner cases here. If certain bad value crash the process, we should try to prevent it from happening.